doc/whats_new.rst

```diff
@@ -61,6 +61,7 @@ Changelog
 
     - Added support for multiclass decoding in :class:`mne.decoding.CSP`, by `Jean-Remi King`_ and `Alexandre Barachant`_
 
+    - Added obtain digitized location of EEG Reference channel when creating it as a new data channel using :func:`mne.io.add_reference_channels` by `Chris Bailey`_
```

Comment:
Mind your grammar. I propose:

     - Adding an EEG reference channel using :func:`mne.io.add_reference_channels` will now use its digitized location from the FIFF file, if present, by `Chris Bailey`_